caffe2/proto/torch.proto

```diff
@@ -174,6 +177,9 @@ message NodeProto {
 
   // A human-readable documentation for this node. Markdown is allowed.
   optional string doc_string = 6;
+
```

Comment:
We could use an additional string field as well, to store the schema of a script module.

Reply:
Actually, we can use op_type to store the schema, and store the node type as one annotation, with name "IR_NODE_TYPE"